src/generic/grid.cpp

```diff
@@ -3873,6 +3879,36 @@ void wxGrid::ProcessRowLabelMouseEvent( wxMouseEvent& event, wxGridRowLabelWindo
     CalcGridWindowUnscrolledPosition(0, event.GetPosition().y,  NULL, &y, gridWindow);
     int row = YToRow( y );
 
+    if ( m_canceledDragging )
+        if ( event.LeftIsDown() )
+            return;
+        else
+            m_canceledDragging = false;
+
+    if ( event.Dragging() && (m_winCapture == rowLabelWin) )
+    {
+        // start auto scrolling at the edges
```

Comment:
This is not really "auto" scrolling if there is no timer involved, as we don't scroll automatically...

Reply:
'auto' removed.